src/test/java/uk/gov/hmcts/reform/wataskmanagementapi/services/TaskManagementServiceTest.java

```diff
@@ -805,7 +807,8 @@ void assignTask_should_throw_role_assignment_verification_exception_when_assigne
             ))
                 .isInstanceOf(RoleAssignmentVerificationException.class)
                 .hasNoCause()
-                .hasMessage("Role Assignment Verification: The request failed the Role Assignment checks performed.");
+                .hasMessage("Role Assignment Verification: "
+                            + "The user being assigned the Task has failed the Role Assignment checks performed.");
 
             verify(camundaService, times(0)).assignTask(any(), any(), anyBoolean());
         }
```

Comment:
the test name is incorrect. Should be for "when_assignee"

Reply:
Fixed